torchci/lib/bot/cliParser.ts

```diff
@@ -56,6 +56,12 @@ mergeOption.add_argument("-l", "--land-checks", {
   "reliability and decrease risk of revert. The tests added are: pull, Lint and trunk. Note " +
   "that periodic is excluded. (EXPERIMENTAL)",
 });
+mergeOption.add_argument("-r", "--rebase", {
+  help: "Rebase the PR to re run checks before merging.  It will accept a branch name and " +
+  "will default to master if not specifiedd.",
```

Comment:
```suggestion
  "will default to master if not specified.",
```
nit, but also if we want to make this public to other repos, we probably want to handle the general default branch

Reply:
did the default branch thing